constants.py

```diff
@@ -0,0 +1,5 @@
+TRAIN_TERMS_FPATH = "cafa-5-protein-function-prediction/Train/train_terms.tsv"
+OBO_FPATH = "cafa-5-protein-function-prediction/Train/go-basic.obo"  
+
+MODEL_ID = "AmelieSchreiber/esm2_t6_8M_finetuned_cafa5_v2"  # Replace with your Hugging Face hub model name
```

Comment:
This is ok for now, but ideally I'd like this value to be modifiable by the environment (through an environment variable) so we can reuse the same API for different versions of the ESM model.

Reply:
I was essentially setting this through an ENV variable to override the default ESM model

```pything
MODEL_ID = "AmelieSchreiber/esm2_t6_8M_finetuned_cafa5_v2"
```

Reply:
Yeah I saw that. I stuck the constants into a file out of habit but here it makes sense to let them be passed through the environment, even though it adds more steps to local use.